code/modules/clothing/spacesuits/rig/suits/engineering.dm

```diff
@@ -2,7 +2,7 @@
 	name = "engineering rig"
 	desc = "A lightweight and flexible armoured rig suit, designed for mining and shipboard engineering."
 	icon_state = "ds_engineering_rig"
-	armor = list(melee = 45, bullet = 60, laser = 60, energy = 25, bomb = 60, bio = 100, rad = 95)
+	armor = list(melee = 45, bullet = 60, laser = 60, energy = 25, bomb = 60, bio = 100, rad = 75)
```

Comment:
The engineering rig was accidentally missed in armor rebalancing work, and is too powerful

lower the first three values to 40, 50, 50